RELEASE_PROCESS.md

```diff
@@ -0,0 +1,135 @@
+land on master between testing and releasing since we typically tag
+HEAD of master.  If something large does land on master, the release
+tag you create should point specifically at the build that you tested.
+The git SHA of the build you are testing can be found in
+`/opt/opscode/version-manifest.json`.
+
+### Preparing for the release
+
+- [ ] Check that omnibus/config/projects/chef-server.rb has the
+  correct version given the type of changes in this release.
+
+- [ ] Double check omnibus/CHANGELOG.md to ensure it includes all
+  included changes. Update as appropriate.
+
+- [ ] Check omnibus/RELEASE_NOTES.md to ensure that it describes the
+  most important user-facing changes in the release. This file should
+  form the basis of the blog post in the following step. Update as
+  appropriate.
+
+- [ ] Write a blog post for blog.chef.io. This requires a wordpress
+  login. If the release contains a fix for a security vulnerability,
+  you MUST tag the post as "security". Save the post as a draft.
+
+### Building and Releasing the Release
+
+- [ ] Tag the chef-server repository with the release version: `git
+  tag -a VERSION_NUMBER`. The first line of the tag message should be
+  the version number. The other lines are up to you. You're the boss!
+  (You can leave them blank).
+
+- [ ] Push the new tag: `git push origin master --tags`.
+
+- [ ] Trigger a release build in Jenkins using the
+  `chef-server-12-trigger-release` trigger.  Use the tag you created
+  above as the GIT_REF parameter.
+
+- [ ] Wait for the pipeline to complete.
+
+- [ ] Use julia to promote the build: `@julia artifactory promote
+  chef-server VERSION from current to stable`.  Please do this in the
+  "Release Services" room.  Once this is done, the release is
+  available to the public via the APT and YUM repositories.
+
+- [ ] Support should already know a release is coming; however, as a
+  courtesy, drop a message in the Customer Support HipChat Channel.
+
+- [ ] In your local checkout of the chef-web-downloads repository,
+generate an update to the download pages using rake:
+
+```
+git co -b YOUR_INITIALS/release-chef-server-VERSION
+export ARTIFACTORY_PASSWORD="CoolPasswordHere"
+rake fetch
+git add data/
+# make sure all the changes are what you expect
+# write a simple commit message
+git commit -v
+git push origin YOUR_INITIALS/release-chef-server-VERSION
+```
+
+- [ ] Open a GitHub pull request for the chef-web-downloads repository
+based on the branch you just created.
+
+- [ ] Have someone review and merge the PR.
+
+- [ ] Once committed to master, an automated process will make the
+  change to the live website; however, it might take a few minutes for
+  the deploy to complete and then a few more for CDN caches to expire.
+
+- [ ] Publish your draft blog post.
+
+- [ ] Send an email to the chef and chef-dev mailing lists announcing
+the release with a link to the blog post.
+
+Chef Server is now released.
```

Comment:
### Post-Release

Increment the patch release number in all `chef-server` version files (e.g. 12.1.2 becomes 12.1.3)

* omnibus/config/projects/chef-server.rb
* src/chef-mover/relx.config
* src/chef-mover/src/mover.app.src
* src/oc_bifrost/apps/bifrost/src/bifrost.app.src
* src/oc_bifrost/rel/reltool.config (probably not, will confirm)
* src/oc_bifrost/relx.config
* src/oc_erchef/rel/reltool.config (probably not, will confirm)
* src/oc_erchef/relx.config
* src/oc_erchef/src/oc_erchef.app.src